emptytheme/assets/alignments-front.css

```
@@ -73,7 +73,7 @@ body {
 
 	.wp-site-blocks .alignfull {
 		transform: translateX(0px);
-		width: 100% + var(--wp--custom--margin--horizontal));
+		width: 100%;
 		max-width: calc(100% + var(--wp--custom--margin--horizontal));
```

Thanks! That looks like it was a typo, since it wasn't wrapped in `calc()` anyway. 😄 

We should do the same to the `max-width`, and then this should be good to go. 

```suggestion
		max-width: 100%;
```